module/product/src/Infrastructure/Grid/AssociatedProductAvailableChildrenGrid.php

```diff
@@ -77,6 +87,18 @@ public function init(GridConfigurationInterface $configuration, Language $langua
         if ($this->bindingAttributes) {
             $this->addBindingColumn($language);
         }
+        $this->addColumn('_links', new LinkColumn('hal', [
+            'delete' => [
+                'privilege' => 'PRODUCT_UPDATE',
```

Comment:
`'privilege' => 'PRODUCT_UPDATE'` to `'method' => Request::METHOD_DELETE` ?

Reply:
we are removing children from product so we update a product